docs/ref/checks.txt

```diff
@@ -120,6 +120,12 @@ upgrading Django.
   :setting:`CSRF_TRUSTED_ORIGINS` setting must start with a scheme (usually
   ``http://`` or ``https://``) but found ``<hostname>``.
 
+Deprecated settings
+-------------------
+
+* **settings.W001**: Checks the Django settings for used but deprecated
+  variables.
```

Comment:
The check as described here in `docs/ref/checks.txt` has to be identical to the one being actually printed in the terminal. This means:
```suggestion
* **settings.W001**: The ``<name>`` setting is deprecated and its use is not recommended.
```